lib/game/game.dart

```diff
@@ -88,6 +74,7 @@ class _GameState extends State<Game> {
                 child: FloatingActionButton(
                   heroTag: "e",
                   onPressed: () {
+                    setSelectedIdentifier("b2");
```

Comment:
to rozumiem, że na szybko, ogarniemy w ramach poruszania się

Reply:
tak